client/src/ui/modules/military/battle-view/Battle.tsx

```diff
@@ -0,0 +1,125 @@
+import { BattleManager } from "@/dojo/modelManager/BattleManager";
+import { BattleType } from "@/dojo/modelManager/types";
+import { ArmyInfo } from "@/hooks/helpers/useArmies";
+import { Realm, Structure } from "@/hooks/helpers/useStructures";
+import Button from "@/ui/elements/Button";
+import { BattleSide, Troops } from "@bibliothecadao/eternum";
+import { ComponentValue } from "@dojoengine/recs";
+import { motion } from "framer-motion";
+import { useState } from "react";
+import { BattleActions } from "./BattleActions";
+import { BattleProgressBar } from "./BattleProgressBar";
+import { BattleSideView } from "./BattleSideView";
+import { LockedResources } from "./LockedResources";
+import { TopScreenView } from "./TopScreenView";
+
+export interface Health {
```

Comment:
should we move this in eternum sdk ?

Reply:
Health will be removed from the `BattleView` so it can be removed